control/statesp.py

```diff
@@ -460,10 +460,8 @@ def freqresp(self, omega):
         if isdtime(self, strict=True):
             dt = timebase(self)
             cmplx_freqs = exp(1.j * omega * dt)
-            if ((omega * dt).any() > pi):
-                warn_message = ("evalfr: frequency evaluation"
-                                " above Nyquist frequency")
-                warnings.warn(warn_message)
+            if (max(omega) * dt > math.pi):
```

Comment:
perhaps ```max(abs(omega))```, to allow for negative omega?

nitpick: outermost parentheses are unnecessary.